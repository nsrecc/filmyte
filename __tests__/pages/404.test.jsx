import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Custom404 from 'pages/404';

describe('Custom 404', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Custom404 />);
    expect(wrapper).toMatchSnapshot();
  });

  it('has h1 element containing 404', () => {
    const wrapper = shallow(<Custom404 />);
    expect(wrapper.find('h1').text()).toMatch('404');
  });
});
