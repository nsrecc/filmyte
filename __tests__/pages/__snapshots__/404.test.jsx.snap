// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom 404 matches snapshot 1`] = `
<div
  className="fzfWrapper"
>
  <h1>
    404
  </h1>
  <h2>
    This page could not be found.
  </h2>
</div>
`;
